std/typecons.d

```diff
@@ -518,6 +472,23 @@ template Tuple(Specs...)
             // This is mostly to make t[n] work.
             alias _Tuple_super this;
         }
+        ///
+        unittest
+        {
+            auto t = tuple(1, " hello ", 2.3);
+            writeln(t);        // Tuple!(int, string, double)(1, " hello ", 2.3)
+            writeln(t.expand); // 1 hello 2.3
+            
+            void takeSeveralTypes(int n, string s, double d) 
+            {
+                //Prints "4 test 5.1"
+                writeln(n, " ", s, " ", d);
```

Comment:
In a unittest, you can use `std.format.format` to verify that the output is actually what the comment claims it is.